appveyor.yml

```diff
@@ -30,7 +53,18 @@ test_script:
     $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\TestsResults.xml))
     # Fail if thre are issues
     if($results.Count -gt 0){throw "ScriptAnalyzer found $($results.Count) issues"}
-   
+    # Activate verbose on Powershell
```

Comment:
I would add an extra space and a comment that mention that the next section is about to perform the integration tests